wardenclyffe/templates/streamlogs/streamlog_list.html

```diff
@@ -58,12 +58,12 @@
                     {% with video=object.video %}
                         {% if video %}
                             <a href="{% url 'video-details' video.pk %}">{{video.title}}</a>
-                        {% else %}
+                        {% else %}{% if '.flv' in object.filename %}
```

Comment:
I think an `elif` clause could achieve the same thing in a clearer way here, but this is fine.